src/schemas/json/workflows.json

```
@@ -271,6 +274,38 @@
           "type": "object"
         }
       ]
+    },
+    "for": {
+      "description": "Iterate over a sequence of numbers or through a collection of data, such as a list or map.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "value": {
+          "description": "Loop variable name.",
+          "type": "string"
+        },
+        "index": {
+          "description": "Index variable name.",
+          "type": "string"
+        },
+        "range": {
+          "description": "DESCRIBE",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 2,
+          "maxItems": 2
+        },
+        "in": {
+          "description": "List expression.",
+          "type": "array"
+        },
+        "steps": {
+          "description": "Each iteration the steps will be executed.",
```

```suggestion
          "description": "Steps for loop\nhttps://cloud.google.com/workflows/docs/reference/syntax/iteration",
```

The same issue as above.